<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>441</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Города</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"><item><widget class="QLabel" name="citiesC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:20px;</string>
     </property>
     <property name="text">
      <string>Записей:</string>
     </property>
    </widget></item>
       <item>
        <widget class="QTableWidget" name="cities">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Типы собственности</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"><item><widget class="QLabel" name="proptypesC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:20px;</string>
     </property>
     <property name="text">
      <string>Записей:</string>
     </property>
    </widget></item>
       <item>
        <widget class="QTableWidget" name="proptypes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Типы продуктов</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"><item><widget class="QLabel" name="prodtypesC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:20px;</string>
     </property>
     <property name="text">
      <string>Записей:</string>
     </property>
    </widget></item>
       <item>
        <widget class="QTableWidget" name="prodtypes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Названия продуктов</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"><item><widget class="QLabel" name="prodnamesC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:20px;</string>
     </property>
     <property name="text">
      <string>Записей:</string>
     </property>
    </widget></item>
       <item>
        <widget class="QTableWidget" name="prodnames">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
